xDrip/Scenes/Entries/EditFoodEntry/EditFoodEntryInteractor.swift

```diff
@@ -11,22 +11,83 @@
 //
 
 import UIKit
+import RealmSwift
```

Comment:
Don't use Realm directly in an interactor. Either move this logic to the Entities themselves or create a separate workers.